src/org/traccar/processing/ComputedAttributesHandler.java

```diff
@@ -96,21 +96,32 @@ protected Position handlePosition(Position position) {
                 } catch (JexlException error) {
                     Log.warning(error);
                 }
+
                 if (result != null) {
                     try {
                         switch (attribute.getType()) {
                             case "number":
-                                position.getAttributes().put(attribute.getAttribute(), (Number) result);
+                                if (attribute.getAttribute().equals(false)) {
```

Comment:
How can it be equal to `false` if it's a number? Am I missing something? Same question for default case as well.

Reply:
Its a catch, in case user returns false then it handles it like null else there may be a cast error trying to convert a boolean to Number.

Reply:
Why would it be boolean value?

Reply:
Im not sure JEXL cares, it will return false/null as the expression is written. Its just a failsafe check in-case the expression is written in that way. More belts and braces approach rather than allow exceptions to happen. For string type if expression returns literal false it would be cast to string, here it tests the literal and handles it while if the user wants text false they should write literal string "false".

Reply:
I don't agree with this approach. If expression returns wrong type it should fail with exception.

Reply:
Ok ile put in exceptions instead.